typescript/web/src/hooks/use-image-navigation.fixtures.ts

```diff
@@ -0,0 +1,81 @@
+import { GET_ALL_IMAGES_OF_A_DATASET_QUERY } from ".";
+import {
+  GetAllImagesOfADatasetQuery,
+  GetAllImagesOfADatasetQueryVariables,
+  GetAllImagesOfADatasetQuery_dataset_images,
+} from "../graphql-types/GetAllImagesOfADatasetQuery";
+import {
+  ApolloMockResponse,
+  ApolloMockResponses,
+} from "../utils/tests/apollo-mock";
+import { BASIC_DATASET_DATA } from "../utils/tests/data.fixtures";
+import { CURRENT_IMAGE } from "./test-utils";
+
+export const IMAGE1 = {
+  id: "929f5445-812c-4679-86bd-0f0748679f38",
+  url: "",
+  thumbnail200Url: "",
```

Comment:
Can we at-least use something which would work?
For example something like `https://labelflow.ai/static/icon-512x512.png`.